tensorboard/backend/event_processing/plugin_event_accumulator.py

```diff
@@ -29,6 +29,9 @@
 from tensorboard.backend.event_processing import io_wrapper
 from tensorboard.backend.event_processing import plugin_asset_util
 from tensorboard.backend.event_processing import reservoir
+from tensorboard.backend.event_processing.event_accumulator import EventAccumulator
```

Comment:
Please remove this import, per https://google.github.io/styleguide/pyguide.html#Imports